OKR/PROJECT.md

```diff
@@ -2,8 +2,27 @@
 
 We frame our ongoing work using a process based on quarterly Objectives and Key Results (OKRs). Objectives reflect outcomes that are challenging, but realistic. Results are tangible and measurable.
 
-## 2018 Q4
+## 2019 Q1
 
+- **The IPFS Org gets a fully dedicated Package Managers Working Group**
+  - `PX` - @??? - A Package Manager Working Group Roadmap is created
+  - `PX` - @??? - Onboard 2+ contributors to this Working Group       
+- **Support and guide the IPFSConf planning and Content Curation**
+  - `PX` - @daviddias - The IPFSConf is scheduled, announced and there is a CFP published   
+  - `PX` - @??? - Working Groups have their goals established for IPFSConf    
+- **The IPFS Working Groups feel supported and they have the information they need to make the best decisions**
+  - `PX` - @??? - Ship the first IPFS User Quarterly Report. It documents achievements and demos of new capabilities
+  - `PX` - @??? - We gather, process and present metrics for the IPFS Project    
+  - `PX` - @??? - We grow the Project WG to have a team capable of answering to all IPFS Working Group needs.
+  - `PX` - @??? - Working group roadmaps are solidified, presented, and used to chart Q1 and Q2 planning
```

Comment:
@momack2 I believe L16 and L17 were the KRs you were highlighting as available to tackle in this comment https://github.com/ipfs/team-mgmt/pull/793#issuecomment-448155532